projects/core/src/occ/adapters/organization/converters/occ-order-approval-decision-normalizer.spec.ts

```diff
@@ -18,11 +21,11 @@ describe('OrderApprovalDecisionNormalizer', () => {
   let service: OccOrderApprovalDecisionNormalizer;
 
   const orderApprovalDecision: Occ.OrderApprovalDecision = {
-    decision: 'APPROVE',
+    decision: Occ.OrderApprovalDecisionValue.APPROVE,
   };
 
   const convertedOrderApprovalDecision: OrderApprovalDecision = {
-    decision: 'APPROVE',
+    decision: OrderApprovalDecisionValue.APPROVE,
```

Comment:
either use the one from occ or from order-approva.model

Reply:
We use both because the one from occ is converted to the other.